### PR TITLE
Apply proposed fixes from the issue list

### PR DESCRIPTION
**Pull request by Felix-Indoing**
Please have a look at each commit message for what is included.

### CODE REVIEW

**Comment by larsdecker** [comment]
@artf 
Can you please look at it and merge it. 

For me the code looks good.

File: src/blocks.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default (editor, opt = {}) => {
 
   bm.add('mj-image', {
     label: 'Image',
-    content: '<mj-image src="http://placehold.it/350x250/78c5d6/fff">',
+    content: '<mj-image src="http://placehold.it/350x250/78c5d6/fff" padding-top="0" padding-right="0" padding-left="0" padding-bottom="0"></mj-image>',
     attributes: { class: 'fa fa-image' },
     ...allBlocks,
   });
```
Comment (artf) [inline src/blocks.js]: Why these changes to paddings?

File: src/components/Section.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ export default (editor, { dc, coreMjmlModel, coreMjmlView }) => {
         draggable: '[data-gjs-type=mj-body]',
         droppable: '[data-gjs-type=mj-column]',
         'style-default': {
-          'padding-top': '10px',
-          'padding-bottom': '10px',
+          // 'padding-top': '10px',
+          // 'padding-bottom': '10px',
```
Comment (artf) [inline src/components/Section.js]: I think those changes in blocks should be placed here, no?

File: src/style.js [hunk 3]
```diff
@@ -72,7 +72,7 @@ export default (editor, opt = {}) => {
           }],
       }, {
         name: 'Decorations',
-        open: false,
+        open: true,
         buildProps: ['background-color', 'container-background-color', 'background-url', 'background-repeat',
           'background-size', 'border-radius', 'border'],
         properties: [{
```
Comment (artf) [inline src/style.js]: Any issue reference to this change? 
IMHO is better to have less UI open, but if this is something highly requested I can accept it otherwise I'd prefer a rollback

File: src/components/Image.js [hunk 4]
```diff
@@ -23,15 +23,45 @@ export default (editor, { dc, coreMjmlModel, coreMjmlView }) => {
...
+        // This will also fix an issue caused by changing void property value
+        // where an imported template has an mj-image that is not enclosed by a closing tag,
+        // the editor adds the mj-image closing tag right before its parent's closing tag.
+        // Ultimately making all its sibling as its children.
+        model.get('components').each(model => {
+          code += model.toHTML();
+        });
+
+        return code;
+      }
```
Comment (DRoet) [inline src/components/Image.js]: I don't believe this is the correct fix for https://github.com/artf/grapesjs-mjml/issues/149 , this seems more like a workaround since `<mj-image />` should be valid mjml. @artf if you have some time I would love to hear your input on this and the linked issue. It seems like the `void` option doesn't get applied correctly in grapesjs when being used with `mj-` components

File: src/style.js (hunk 3 above)
Comment (larsdecker) [reply src/style.js]: For me, i think it is better that the default is close.
I prefer the rollback for this.

File: src/blocks.js (hunk 1 above)
Comment (Felix-Indoing) [reply src/blocks.js]: The default style is incorrect if we don't add the padding zero by default.
image

File: src/components/Section.js (hunk 2 above)
Comment (Felix-Indoing) [reply src/components/Section.js]: For this one, it actually comes from one of my users complained that he cannot get a true WYSIWYG experience because there was always padding between sections that cannot be disabled. I think it will be better to provide a UI option to toggle it, but I haven't got time to do so.

File: src/style.js (hunk 3 above)
Comment (Felix-Indoing) [reply src/style.js]: This one also come from one of my users that complained too many clicks required to change styles. The default to close is okay.

**Comment by DRoet** [comment]
for easier review/merge this should probably be split into several PRs, I cherry picked one commit from this PR into https://github.com/artf/grapesjs-mjml/pull/171

**Comment by DRoet** [comment]
closing this due to inactivity, please let me know if you still want to work on this
